cilium/cmd/bpf_mountfs_show.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2020 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/cilium/cilium/pkg/mountinfo"
+	"github.com/spf13/cobra"
+)
+
+// bpfmountfsShowCmd represents the bpf file system show command
+var bpfmountfsShowCmd = &cobra.Command{
+	Use:     "show",
+	Short:   "Show bpf filesystem mount details",
+	Example: "cilium bpf fs show",
+	Run: func(cmd *cobra.Command, args []string) {
+		getbpfmountFS(cmd, args)
+	},
+}
+
+func init() {
+	bpfmountFSCmd.AddCommand(bpfmountfsShowCmd)
```

Comment:
Can we add JSON support too? For examples, see how the other CLI commands do this; it should be pretty simple.

Reply:
Sure will add it.

Reply:
Done.